charts/mlp/templates/_helpers.tpl

```diff
@@ -27,8 +27,12 @@ Generated names
     {{- end -}}
 {{- end -}}
 
+{{- define "mlp.config-cm-name" -}}
+    {{- printf "%s-config" (include "mlp.name" .) | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
 {{- define "mlp.encryption-key-name" -}}
-    {{- printf "%s-encryption-key" (include "mlp.resource-prefix" .) | trunc 63 | trimSuffix "-" -}}
+    {{- printf "%s-encryption-key" (include "mlp.name" .) | trunc 63 | trimSuffix "-" -}}
```

Comment:
could we stick to using `mlp.resource-prefix` here? also in the cm above

Reply:
The problem with using `mlp.resource-prefix` is that it ignores `fullnameOverride` and `nameOverride`  values, so the resulting resource's name (ConfigMap/Secret) is `mlp-0-4-0-config` or something like. Don't you think it should be consistent with `Deployment/Service` and to take into account the overrides?

Reply:
agree, I can work on making this consistent in another PR later.